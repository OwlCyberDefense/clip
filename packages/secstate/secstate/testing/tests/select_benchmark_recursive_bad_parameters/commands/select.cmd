/usr/sbin/secstate select password-req -j
