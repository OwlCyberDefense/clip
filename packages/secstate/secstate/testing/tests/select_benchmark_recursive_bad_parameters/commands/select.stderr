usage: secstate select [options] <ContentID> [GroupID|RuleID|ProfileID]

secstate: error: no such option: -j
